Package.swift

```diff
@@ -15,12 +15,16 @@ let package = Package(
         .library(name: "SwifterSwift", targets: ["SwifterSwift"])
     ],
     targets: [
-        .target(name: "SwifterSwift",
-                swiftSettings: [
-                    .enableUpcomingFeature("ConciseMagicFile"),
-                    .enableUpcomingFeature("ExistentialAny"),
-                    .enableUpcomingFeature("ForwardTrailingClosures")
-                ]),
+        .target(
+            name: "SwifterSwift",
+            resources: [
+                .process("Resources/PrivacyInfo.xcprivacy")
+            ],
+            swiftSettings: [
```

Comment:
Hello @guykogus 
Are these `swiftSettings` required for a private manifest? Just asking because I also need to add it to my app.

Reply:
No, not at all, they were just always here so I haven't really looked much into them.